src/index.js

```diff
@@ -0,0 +1,136 @@
+module.exports = {
+  root: true
```

Comment:
```suggestion
  root: true,
```
désolé j'ai oublier la virgule